src/icons/badges/MetadataDefaultBadge.js

```diff
@@ -20,7 +19,7 @@ const MetadataDefaultBadge = ({ className = '', height = 16, title, width = 16 }
         width={width}
     >
         <g fill="none" fillRule="evenodd">
-            <circle cx="8" cy="8" fill={bdlNeutral04} r="7.5" stroke="#767676" />
+            <circle cx="8" cy="8" fill="none" r="7.5" stroke="#767676" />
```

Comment:
Since you're here can we use the variables for the stroke? Looks like `#767676` exists in `_variables.scss` but can the one from `MetadataActiveBadge` use a similar existing color?